app/frontend/src/features/messages/messagelist/MessageList.tsx

```diff
@@ -1,81 +1,38 @@
 import { Box, BoxProps } from "@material-ui/core";
 import { makeStyles } from "@material-ui/core/styles";
 import * as React from "react";
-import { useState } from "react";
-import { useForm } from "react-hook-form";
-import Alert from "../../../components/Alert";
-import Button from "../../../components/Button";
-import TextField from "../../../components/TextField";
 import { Message } from "../../../pb/conversations_pb";
-import MessageView from "./Message";
+import { isControlMessage } from "../utils";
+import ControlMessageView from "./ControlMessageView";
+import MessageView from "./MessageView";
 
 const useStyles = makeStyles({
-  root: {},
   list: {
     display: "flex",
     flexDirection: "column-reverse",
   },
 });
 
-interface MessageFormData {
-  text: string;
-}
-
 export interface MessageListProps extends BoxProps {
   messages: Array<Message.AsObject>;
-  handleSend: (text: string) => void;
 }
 
-export default function MessageList({
-  messages,
-  handleSend,
-}: MessageListProps) {
-  const [error, setError] = useState("");
-  const { register, handleSubmit } = useForm<MessageFormData>();
+export default function MessageList({ messages }: MessageListProps) {
   const classes = useStyles();
 
-  const onSubmit = handleSubmit(async (data: MessageFormData) => {
-    setError("");
-    try {
-      await handleSend(data.text);
-    } catch (error) {
-      setError(error.message);
-    }
-  });
-
   return (
-    <Box className={classes.root}>
-      {error && <Alert severity="error">{error}</Alert>}
-
-      <Box className={classes.list}>
-        {messages.length ? (
-          messages.map((message) => (
+    <Box className={classes.list}>
+      {messages.length ? (
+        messages.map((message) =>
+          isControlMessage(message) ? (
+            <ControlMessageView key={message.messageId} message={message} />
+          ) : (
             <MessageView key={message.messageId} message={message} />
-          ))
-        ) : (
-          <>No messages</>
-        )}
-      </Box>
-
-      <form onSubmit={onSubmit}>
-        <TextField
-          label="Text"
-          name="text"
-          defaultValue={""}
-          inputRef={register}
-          rowsMax={5}
-          multiline
-        />
-
-        <Button
-          type="submit"
-          variant="contained"
-          color="primary"
-          onClick={onSubmit}
-        >
-          Send
-        </Button>
-      </form>
+          )
+        )
+      ) : (
+        <>No messages</>
```

Comment:
Shouldn't this be wrapped in `Typography` or at least a `<p>` tag rather than the text on its own?